engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OAuth2Configuration.scala

```diff
@@ -79,7 +78,8 @@ object JwtConfiguration {
 
   implicit val jwtConfigurationVR: ValueReader[JwtConfiguration] = ValueReader.relative(_.rootAs[JwtConfig])
 
-  private case class JwtConfig(enabled: Boolean = false,
+  private case class JwtConfig(accessTokenIsJwt: Boolean = false,
```

Comment:
Defaults on class is ugly way.. Better is create object with `apply(publicKey: Option[String],publicKeyFile: Option[String], certificate: Option[String])` which creates `JwtConfig(accessTokenIsJwt = false, userinfoFromIdToken = false, publicKey, publicKeyFile, certificate)`.

Reply:
This is a configuration class for Typesafe Config. Will it handle `apply`? No such pattern in the existing code.

Reply:
And it is a `case class` which is already Scala's syntactic sugar for the `class` and `apply` in a companion object.